web/server/codechecker_server/api/report_server.py

```diff
@@ -2534,20 +2553,38 @@ def getLinesInSourceFileContents(self, lines_in_files_requested, encoding):
         self.__require_view()
         with DBSession(self._Session) as session:
             res = defaultdict(lambda: defaultdict(str))
-            for lines_in_file in lines_in_files_requested:
-                if lines_in_file.fileId is None:
-                    LOG.warning("File content requested without a fileId.")
-                    LOG.warning(lines_in_file)
-                    continue
-                sourcefile = session.query(File).get(lines_in_file.fileId)
-                cont = session.query(FileContent).get(sourcefile.content_hash)
-                lines = zlib.decompress(
-                    cont.content).decode('utf-8', 'ignore').split('\n')
-                for line in lines_in_file.lines:
+
+            # This will contain all the lines for the given fileId
+            contents_to_file_id = defaultdict(lambda: list())
```

Comment:
```suggestion
            contents_to_file_id = defaultdict(list)
```
`list` in itself is a callable.

Reply:
Yep, forgot this one.